paddle/fluid/framework/ir/mkldnn/mkldnn_fc_rnn_fuse_pass_tester.cc

```diff
@@ -23,6 +23,20 @@ namespace paddle {
 namespace framework {
 namespace ir {
 
+void RegisterOpKernels() {
```

Comment:
Maybe this could be put in pass_tester_helper.h and used in both test files. It could accept something like const std::vector<string>& with names of ops that need to be registered.

Reply:
Why does this function not use static variable and this one does?
https://github.com/PaddlePaddle/Paddle/pull/49899/files#diff-e89175d68effdff4937805b6a1756de1ccb7592a23c69d25d00d4780e6e545ecR28

Reply:
Good idea, I threw it into pass_tester_helper.h